wordpress-mobile/WordPress-iOS
# Remove Stories and Kanvas dependency

The "Stories" feature was [decommissioned](https://github.com/wordpress-mobile/WordPress-iOS/pull/22449) five months ago: you can no longer created, view, or edit stories natively in the app. This PR removes the remaining code.

- Reduces the universal .ipa size by another **2.7 MB**
- Removes the `Kanvas` dependency (25K of Swift) – one less pod to deal with

## To test:

- Open the app and verify that it still runs
- Open "Site Media" and verify that the cells are displayed

## Regression Notes
1. Potential unintended areas of impact: n/a
2. What I did to test those areas of impact (or what existing automated tests I relied on): n/a


3. What automated tests I added (or what prevented me from doing so)

PR submission checklist:


Testing checklist:
